common/vulkan_wrapper.cpp

```diff
@@ -579,6 +579,11 @@ int InitVulkan(void) {
 #endif
 
 #ifdef VK_ENABLE_BETA_EXTENSIONS
+    vkGetPhysicalDeviceVideoEncodeQualityLevelPropertiesKHR =
+        reinterpret_cast<PFN_vkGetPhysicalDeviceVideoEncodeQualityLevelPropertiesKHR>(
+            dlsym(libvulkan, "vkGetPhysicalDeviceVideoEncodeQualityLevelPropertiesKHR"));
+    vkGetEncodedVideoSessionParametersKHR =
+        reinterpret_cast<PFN_vkGetEncodedVideoSessionParametersKHR>(dlsym(libvulkan, "vkGetEncodedVideoSessionParametersKHR"));
```

Comment:
this looks funky - why are we dlsym'ing extension functions? (or has this always been like this and I never noticed)

Reply:
I believe it has always been this way. vulkan_wrapper is an entry point loader that came from the Android folks, but I'm not sure who actually uses it.

Reply:
Ah, well I'll resolve this comment then. Seems very suspicious. This would return nullptr for any desktop users of this code. Still, if its from android, then I can't claim its "wrong". (they do have their own loader after all)